src/Compilers/CSharp/Portable/Symbols/FunctionPointerTypeSymbol.cs

```diff
@@ -0,0 +1,264 @@
+        internal override NamedTypeSymbol? BaseTypeNoUseSiteDiagnostics => null;
+        internal override ManagedKind ManagedKind => ManagedKind.Unmanaged;
+        internal override ObsoleteAttributeData? ObsoleteAttributeData => null;
+        public override void Accept(CSharpSymbolVisitor visitor) => visitor.VisitFunctionPointerType(this);
+        public override TResult Accept<TResult>(CSharpSymbolVisitor<TResult> visitor) => visitor.VisitFunctionPointerType(this);
+        public override ImmutableArray<Symbol> GetMembers() => ImmutableArray<Symbol>.Empty;
+        public override ImmutableArray<Symbol> GetMembers(string name) => ImmutableArray<Symbol>.Empty;
+        public override ImmutableArray<NamedTypeSymbol> GetTypeMembers() => ImmutableArray<NamedTypeSymbol>.Empty;
+        public override ImmutableArray<NamedTypeSymbol> GetTypeMembers(string name) => ImmutableArray<NamedTypeSymbol>.Empty;
+        internal override TResult Accept<TArgument, TResult>(CSharpSymbolVisitor<TArgument, TResult> visitor, TArgument a) => visitor.VisitFunctionPointerType(this, a);
+
+        protected override ISymbol CreateISymbol()
+        {
+            // PROTOTYPE(func-ptr): Implement
+            throw new NotImplementedException();
+        }
+
+        protected override ITypeSymbol CreateITypeSymbol(CodeAnalysis.NullableAnnotation nullableAnnotation)
+        {
+            // PROTOTYPE(func-ptr): Implement
+            throw new NotImplementedException();
+        }
+
+
+        internal override void AddNullableTransforms(ArrayBuilder<byte> transforms)
+        {
+            // PROTOTYPE(func-ptr): Implement
+        }
+
+        internal override bool ApplyNullableTransforms(byte defaultTransformFlag, ImmutableArray<byte> transforms, ref int position, out TypeSymbol result)
+        {
+            // PROTOTYPE(func-ptr): Implement
+            result = this;
+            return true;
+        }
+
+        internal override bool GetUnificationUseSiteDiagnosticRecursive(ref DiagnosticInfo result, Symbol owner, ref HashSet<TypeSymbol> checkedTypes)
+        {
+            throw new NotImplementedException();
+        }
+
+        internal override ImmutableArray<NamedTypeSymbol> InterfacesNoUseSiteDiagnostics(ConsList<TypeSymbol>? basesBeingResolved = null)
+        {
+            throw new NotImplementedException();
+        }
+
+        internal override TypeSymbol MergeEquivalentTypes(TypeSymbol other, VarianceKind variance)
+        {
+            throw new NotImplementedException();
+        }
+
+        internal override TypeSymbol SetNullabilityForReferenceTypes(Func<TypeWithAnnotations, TypeWithAnnotations> transform)
+        {
+            throw new NotImplementedException();
+        }
+
+        // PROTOTYPE(func-ptr): Make source and metadata versions
+        private sealed class FunctionPointerMethodSymbol : MethodSymbol
+        {
+            private readonly FunctionPointerTypeSymbol _parent;
```

Comment:
>private readonly FunctionPointerTypeSymbol _parent; [](start = 12, length = 51)

It feels like ownership shouldn't matter. Similar as there is no ownership relationship between a PointerTypeSymbol and its underlying type. This is not a member.